.github/workflows/ci.yml

```diff
@@ -195,45 +188,46 @@ jobs:
             --env "SUITE_NAME=${{ matrix.suite.name }}" \
             --env "CI_NODE_TOTAL=${{ matrix.suite.ci_node_total }}" \
             --env "CI_NODE_INDEX=${{ matrix.suite.ci_node_index }}" \
-            --rm dependabot-updater bash -c \
+            --rm dependabot/dependabot-updater-${{ matrix.suite.ecosystem }} bash -c \
             "cd /home/dependabot/${{ matrix.suite.path }} && ./script/ci-test"
 
   updater:
+    env:
+      DOCKER_BUILDKIT: 1
     name: Updater
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-      - name: Build dependabot-core image
-        env:
-          DOCKER_BUILDKIT: 1
+      - name: Build updater core image
         run: |
           docker build \
-            -t "dependabot/dependabot-core:latest" \
+            -t "ghcr.io/dependabot/dependabot-updater-core:latest" \
             --build-arg BUILDKIT_INLINE_CACHE=1 \
-            --cache-from ghcr.io/dependabot/dependabot-core \
+            --cache-from ghcr.io/dependabot/dependabot-updater-core \
+            -f Dockerfile.updater-core \
             .
-      - name: Build dependabot-updater image
-        env:
-          DOCKER_BUILDKIT: 1
+      # using bundler as the test updater
+      - name: Build ecosystem image
         run: |
           docker build \
-            -f Dockerfile.updater \
-            -t "dependabot-updater:latest" \
-            --build-arg OMNIBUS_VERSION=latest \
+            -t "dependabot/dependabot-updater-bundler:latest" \
             --build-arg BUILDKIT_INLINE_CACHE=1 \
-            --cache-from ghcr.io/dependabot/dependabot-updater \
+            --cache-from ghcr.io/dependabot/dependabot-updater-bundler \
+            -f bundler/Dockerfile \
             .
       - name: Run updater tests
+        env:
+          DEPENDABOT_TEST_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           docker run --env DEPENDABOT_TEST_ACCESS_TOKEN \
             --env VCR \
             --rm \
             -v "$(pwd)/updater/spec/fixtures/vcr_cassettes:/home/dependabot/dependabot-updater/spec/fixtures/vcr_cassettes" \
-            "dependabot-updater:latest" bundle exec rspec
-        env:
-          SKIP_BUILD: true
-          DEPENDABOT_TEST_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            -v "$(pwd)/npm_and_yarn:/home/dependabot/npm_and_yarn" \
+            -v "$(pwd)/go_modules:/home/dependabot/go_modules" \
+            -v "$(pwd)/composer:/home/dependabot/composer" \
+            "dependabot/dependabot-updater-bundler:latest" bundle exec rspec
```

Comment:
Can you add a code comment clarifying why _these_ four ecosystems in particular have to be mounted in here? I know why, but if I lacked that context from other PR's, it'd be confusing why these and not any others...